ibm/service/power/resource_ibm_pi_instance_test.go

```diff
@@ -191,6 +225,25 @@ func TestAccIBMPIInstanceBasic(t *testing.T) {
 	})
 }
 
+func TestAccIBMPIInstanceDeploymentType(t *testing.T) {
```

Comment:
Can we re-use existing test case and include optional deployment_type via environment variable? This way we don't need to add new cases for every change.